security/.htaccess

```diff
@@ -5,3 +5,4 @@ RewriteEngine on
 RewriteRule ^$ https://w3c-ccg.github.io/security-vocab/ [R=302,L]
 RewriteRule ^v1$ https://w3c-ccg.github.io/security-vocab/contexts/security-v1.jsonld [R=302,L]
 RewriteRule ^v2$ https://w3c-ccg.github.io/security-vocab/contexts/security-v2.jsonld [R=302,L]
+RewriteRule ^v3$ https://w3c-ccg.github.io/security-vocab/contexts/security-v3-unstable.jsonld [R=302,L]
```

Comment:
We should name this v3-unstable... if we don't, people will ship with an experimental context and blame us.

Reply:
@msporny yep I agree... where do we write that decision down? 

something to the effect of "URLs that contain unstable may have their resources altered without notice".

"URLs that do not contain unstable will never be updated.

Reply:
I have updated the unstable URL, it will be:

`https://w3id.org/security/v3-unstable`